README.md

```diff
@@ -96,10 +102,11 @@ Available Commands:
 Use "pick [command] --help" for more information about a command.
 ```
 
-## Threat model
+## Security
+
+While `pick` is focused on security, vulnerabilities or backdoors in hardware, including hard drives, memory chips, CPU or peripherals are outside of our threat model.
 
-Although `pick` is focused on security, once an adversary somehow gains write access to your computer where the `pick` binary is stored, he can simply exchange it and record your master password to decrypt your pick safe.
```

Comment:
I find these two lines pretty important (101, 102) and would revert the change / rephrase these sentences instead.

Reply:
I was attempting to declare what is outside of our threat model, rather than describing an example attack. Do you have any suggestions on how to communicate this?

Reply:
We should bring to mind that it's crucial to only use the `pick` binary on a trusted PC. The PC storing the pick safe doesn't necessarily need to be trusted (e.g. `Amazon S3`).

What do you think of the following text (replaces the new sentence ```While `pick` is focused on security, vulnerabilities or backdoors in hardware [..]```):
```
`pick` is focussed on security. It is _essential_ to only run the `pick` binary on a
trusted computer. You don't necessarily need to trust the computer or server storing
the pick safe (e.g. `Amazon S3`). As the pick safe is encrypted and authenticated,
no one can decrypt or unnoticeably modify it.
```

Reply:
I took the above and ever so slightly modified it-- thoughts?

Reply:
😻 Awesome! One more thing to make this absolutely perfect — sorry:
The new `## Security` section and some of the new texts under `## Getting started` now use manual line wrapping. Looks a bit strange when reading the `README` file uninterpretedly.

Reply:
Bah, yeah that's my editor adding line breaks. I'll fix that.

Reply:
re: the line breaks- I've updated the README so that lines are wrapped at 80 cols, this way
it's possible to read the file without rendering.